tests/integration/test_integ.py

```diff
@@ -0,0 +1,129 @@
+# HOW TO EXECUTE?
+# First install the package: pip install pytest-docker-compose
+# Run with: pytest --docker-compose="path_to_docker_compose_yml" test_integ.py
+
+import pytest
+import os
+import subprocess
+import unittest
+import time
+import sys
+import logging
+
+
+class TestIntegratedTool(unittest.TestCase):
+
+
+    def setUp(self):
+        print("/************************************* Entered setUp *************************************/")
+
+        self.mounted_files = []
+        self.FUSE_MOUNTED_FILES_LIST = "/tmp/Mounted_files.lst"
+        self.MDH_CORE = "core-sfs"
+        self.MOUNTED_FROM = "/home/sfsuser/Vaidehi/amos-rep/test_tree"
+
+
+        # Check if mdh core is running
+        is_core_up = subprocess.Popen(["docker", "container", "inspect", "-f",
+	                              "'{{.State.Running}}'", self.MDH_CORE], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        out, err = is_core_up.communicate()
+        if "true" not in out.decode():
+            print("MDH CORE NOT RUNNING!!!")
+            exit()
+
+        docker_compose_up=subprocess.Popen(["docker-compose", "up"], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
i think that would be the single place where you must use `subprocess.Popen` or just `subprocess.run` for managing containers. i tried it with `["docker-compose", "up", "-d", "--build"]` and it returned as it finished.

Reply:
changed to the mentioned command